fpdf/fpdf.py

```diff
@@ -3427,6 +3433,8 @@ def _render_styled_text_line(
                         self._resource_catalog.add(
                             PDFResourceType.FONT, current_font.i, self.page
                         )
+                else:
+                    self._set_font_size_for_page()
```

Comment:
Shouldn't `self._set_font_size_for_page()` also be called in the block above, if the `if` condition is `True`?

Reply:
Handled in [`1b48d3e` (#1387)](https://github.com/py-pdf/fpdf2/pull/1387/commits/1b48d3e4a2594b96aa6339daef0a6b364a5cbdfe)